Wrappers/Python/cil/processors/Masker.py

```diff
@@ -0,0 +1,168 @@
+# -*- coding: utf-8 -*-
+#   This work is part of the Core Imaging Library (CIL) developed by CCPi 
+#   (Collaborative Computational Project in Tomographic Imaging), with 
+#   substantial contributions by UKRI-STFC and University of Manchester.
+
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+
+#   http://www.apache.org/licenses/LICENSE-2.0
+
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+from cil.framework import DataProcessor, AcquisitionData, ImageData, ImageGeometry, DataContainer
+import warnings
+import numpy
+
+class Masker(DataProcessor):
+    r'''
+    Processor to fill missing values provided by mask. Please use the desiried method to configure a processor for your needs.
+    '''
+
+    @staticmethod
+    def value(mask=None, value=0):
+        r'''This imputes value where mask == 0.
+        :param mask: mask 
+        :type mask: DataContainer
```

Comment:
I think this could be any Derived DataContainer too, and maybe we could add numpy arrays without much effort.